src/content/configuration/dev-server.md

```diff
@@ -816,7 +816,7 @@ module.exports = {
 
 ## `devServer.open`
 
-`boolean = false` `string`
+`boolean = false` `string` `object`
```

Comment:
https://webpack.js.org/contribute/writers-guide/#configuration-defaults-and-types
```suggestion
`boolean = false` `string` `object = { app [string] }`
```